batched/dense/unit_test/Test_Batched_SerialGetrf.hpp

```diff
@@ -0,0 +1,369 @@
+//@HEADER
+// ************************************************************************
+//
+//                        Kokkos v. 4.0
+//       Copyright (2022) National Technology & Engineering
+//               Solutions of Sandia, LLC (NTESS).
+//
+// Under the terms of Contract DE-NA0003525 with NTESS,
+// the U.S. Government retains certain rights in this software.
+//
+// Part of Kokkos, under the Apache License v2.0 with LLVM Exceptions.
+// See https://kokkos.org/LICENSE for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//@HEADER
+/// \author Yuuichi Asahi (yuuichi.asahi@cea.fr)
+#include <gtest/gtest.h>
+#include <Kokkos_Core.hpp>
+#include <Kokkos_Random.hpp>
+#include <KokkosBatched_Util.hpp>
+#include <KokkosBatched_Getrf.hpp>
+#include <KokkosBatched_Gemm_Decl.hpp>
+#include "Test_Batched_DenseUtils.hpp"
+
+using namespace KokkosBatched;
```

Comment:
Since these headers get included into a big test file, I'd rather not pollute the global namespace.

Reply:
Makes sense. I have fixed accordingly.